freeipa.spec.in

```diff
@@ -398,6 +398,7 @@ BuildRequires:  python3-pylint
 BuildRequires:  python3-pytest-multihost
 BuildRequires:  python3-pytest-sourceorder
 BuildRequires:  python3-qrcode-core >= 5.0.0
+BuildRequires:  python3-pyOpenSSL
```

Comment:
Do we need it as build dependency?

Reply:
We don't. This is a leftover from when I had the import at the top of the file instead of a lazy import. I'll rebase and remove it.

Reply:
Adding the BR back in. If CI lint passes then yes, it's required.